our-app/src/components/unauthenticated/HomePage.js

```diff
@@ -1,9 +1,82 @@
 import React from 'react';
+import { Carousel } from 'react-bootstrap';
 
-function HomePage() {
+export default function HomePage() {
+    const mystyle = {
+        color: "Black",
+        fontFamily: "Helvetica"
```

Comment:
Need to talk to Justin to make sure we are keeping the font consistent...

Reply:
All current styling comes from index.css

Reply:
True! I had to change it for homepage because I changed the background color but I totally get what you mean. I think we should finalize color so that we remain consistent.